recipes/libastra/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set version = "1.9.9" %}
+{% set build = 0 %}
+
+package:
+  name: libastra
+  version: {{ version }}
+
+source:
+  url: https://github.com/tomopy/astra-toolbox/archive/refs/tags/{{ version }}.tar.gz
+  sha256: fc8c1bc2553cd93e570e1991b44df301c89ce7d9521033672f2d9d421af94beb
+
+build:
```

As this is a library, this should also have a `run_exports`.

---

I think `cudatoolkit` and the other runtime libraries are automatically added when the compilers are added to the `requirements/build`?

---

No, `run_exports` is about what pinning should be added to `run` if `libastra` appears as a direct dependency in `hosts`. If there are no ABI guarantees (I didn't see anything about this in the README), this should be:
```suggestion
build:
  run_exports:
    - {{ pin_subpackage('libastra', max_pin='x.x.x') }}
```

---

Thanks for the explanation! 😄 You're right. I don't really trust the main devs to follow semantic versioning.